packages/contracts-bedrock/scripts/upgrades/holocene/scripts/common.sh

```diff
@@ -71,61 +70,33 @@ prompt() {
 #   - The release version is not found in the TOML file
 #   - The implementation address for the specified contract is not found
 fetch_standard_address() {
-    local network_name="$1"
-    local release_version="$2"
-    local contract_name="$3"
-
-    # Determine the correct toml url
-    local toml_url="https://raw.githubusercontent.com/ethereum-optimism/superchain-registry/refs/heads/main/validation/standard/standard-versions"
-    if [ "$network_name" = "mainnet" ]; then
-        toml_url="$toml_url-mainnet.toml"
-    elif [ "$network_name" = "sepolia" ]; then
-        toml_url="$toml_url-sepolia.toml"
-    else
-        echo "Error: NETWORK must be set to 'mainnet' or 'sepolia'"
-        exit 1
-    fi
+  local network_name="$1"
+  local release_version="$2"
+  local contract_name="$3"
 
-    # Fetch the TOML file content from the URL if not already cached for this URL
-    if [ -z "${CACHED_TOML_CONTENT[$toml_url]:-}" ]; then
-        CACHED_TOML_CONTENT[$toml_url]=$(curl -s "$toml_url")
-        # shellcheck disable=SC2181
-        if [ $? -ne 0 ]; then
-            echo "Error: Failed to fetch TOML file from $toml_url"
-            exit 1
-        fi
-    fi
+  if [[ "$network_name" != "mainnet" && "$network_name" != "sepolia" ]]; then
+    echo "Error: NETWORK must be set to 'mainnet' or 'sepolia'"
+    exit 1
+  fi
 
-    # Use the cached content for the current URL
-    local toml_content="${CACHED_TOML_CONTENT[$toml_url]}"
+  # Ensure cache dir exists
+  CACHE_DIR="${CACHE_DIR:-$(mktemp -d)}"
 
-    # Find the section for v1.6.0 release
-    # shellcheck disable=SC2155
-    local section_content=$(echo "$toml_content" | awk -v version="$release_version" '
-        $0 ~ "^\\[releases.\"op-contracts/v" version "\"\\]" {
-            flag=1;
-            next
-        }
-        flag && /^\[/ {
-            exit
-        }
-        flag {
-            print
-        }
-    ')
-    if [ -z "$section_content" ]; then
-        echo "Error: v$release_version release section not found in addresses TOML"
-        exit 1
+  local toml_path="${CACHE_DIR}/standard-versions-$network_name.toml"
+  if [[ ! -f "$toml_path" ]]; then
+    local toml_url="https://raw.githubusercontent.com/ethereum-optimism/superchain-registry/refs/heads/main/validation/standard/standard-versions-$network_name.toml"
```

Comment:
I'm wondering about whether we should use a permalink here or not. Thoughts?

Reply:
Hmm, since we already reference a specific tag, it feels like the info in that tag should never change once written. And if it does change, it was probably a critical fix, so it would be ok that this fix is immediately picked up by script runners.

This also has the advantage that if we add additional entries, e.g. for `rc.4`, these can already be used by the script. The script allows for configuration of the op-contracts release via the new `OP_CONTRACT_RELEASES` `.env` var, so this makes the script forwards-compatible to additions. 

So I think it's fine to use a non-permalink and get the latest instead. I also just copied the old URL. That's how it was before.